vars/main.yml

```diff
@@ -10,13 +10,19 @@ pihole_default_packages_apt:
   - 'whiptail'
   - 'ca-certificates'
   - 'sqlite3'
+  - 'apt-utils'
```

Comment:
I believe these changes are already commented on in a another pull request.

Reply:
commented below